rb/spec/unit/selenium/webdriver/chrome/driver_spec.rb

```diff
@@ -54,7 +54,6 @@ module Chrome
         it 'does not set the chrome.detach capability by default' do
           Driver.new(http_client: http)
 
-          expect(caps['goog:chromeOptions']).to eq({})
```

Comment:
This spec can be modified, giving you extra strength (Check this fetch key doesn't work and therefore returns `nil`)

Reply:
I added separate spec to check `goog:chromeOptions` https://github.com/SeleniumHQ/selenium/pull/6078/files#diff-0396c977ce3bbece63a6b7dd4b3ef3c7R54

Reply:
Agreed this is better